.travis.yml

```diff
@@ -2,17 +2,43 @@ language: csharp
 mono: none
 solution: EFCore-Adapter.sln
 dist: xenial
-dotnet: 3.1
+dotnet: 
+    - 3.1
+
+env: 
+    global:
+        - REPO_COMMIT_AUTHOR=$(git show -s --pretty=format:"%cn")
+        - REPO_COMMIT_AUTHOR_EMAIL=$(git show -s --pretty=format:"%ce")
+        - REPO_COMMIT_MESSAGE=$(git show -s --pretty=format:"%s")
+        - DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
+        - DOTNET_CLI_TELEMETRY_OPTOUT=true
+        - BUILD_CONFIGURATION=Release
+
+install:
+    - sudo apt-get update
+    - sudo apt-get install apt-transport-https
```

Comment:
What's this package used for?

Reply:
It looks to be used for apt-get HTTPS support, It is referred from [dotnet core install docs](https://docs.microsoft.com/en-gb/dotnet/core/install/linux-package-manager-ubuntu-1604).